### ISSUE TITLE
Wishlist: Have Auto DJ indicator become a "enable/disable" trigger

### ISSUE THREAD

**Issue by nPrevail** [mixxxdj/mixxx#13511]
### Feature Description

![Make a clickable outline of   auto DJ](https://github.com/user-attachments/assets/5cfc00df-59bb-4f5a-b9a8-db1aaaf62b53)

I'd like to request to have the "Auto DJ icon," located at the top right area of the Mixxx UI, to not only be an indicator of auto DJ turning on/off, but also a trigger than can literally turn the auto DJ feature on and off.

When Auto DJ is off, we could have a "dark silhouette" represent a "off Auto DJ" trigger, and when clicked, it'll illuminate to the normal orange color to indicate "on Auto DJ."

This would save people from having to look at their File Tree -> Auto DJ - > click "I/O Auto DJ" icon.

**Comment by vespadj** [comment]
@nPrevail , like this?
https://github.com/mixxxdj/mixxx/pull/12890#issuecomment-1994130969

**Comment by nPrevail** [comment]
> @nPrevail , like this? [#12890 (comment)](https://github.com/mixxxdj/mixxx/pull/12890#issuecomment-1994130969)

Yes! This would be perfect! Although, I don't think we need a "I/O" symbol for either AutoDJ toggle. It's not like other modules have it either, such as "record" and "broadcast."

I think it's pretty self-explained when it's "on" and "off."

**Comment by jasmeen04** [comment]
I want to solve this issue . Please assign me.

**Comment by ronso0** [comment]
We don't assign issues, any contributor may simply start workin on it (ideally leaving a note).

**Comment by urek69mazino** [comment]
in which file will i find the auto dj icon elment code to edit , for eg in the late night skin , which XML file should i look for ?

**Comment by ronso0** [comment]
In LateNight this is here
https://github.com/mixxxdj/mixxx/blob/edae0ca3b2fc36c51d9f310f341fb2e4bbea7d26/res/skins/LateNight/toolbar.xml#L123
It's a StatusLight which would need to be turned into a 2-state pushbutton (see other button templates in this file for example).

In other skins just look for indicators assigned to `[AutoDJ],enabled`.
